bolt/include/bolt/Core/MCPlusBuilder.h

```diff
@@ -2041,10 +2041,10 @@ class MCPlusBuilder {
     return InstructionListType();
   }
 
-  virtual InstructionListType createDummyReturnFunction(MCContext *Ctx) const {
```

Comment:
Basically you're moving the definition of `createDummyReturnFunction` from X86 to MCPlusBuilder, right? Please avoid renaming the method (I don't see what's achieved by it), and move the definition here so it's clear why we don't stub it with llvm_unreachable

Reply:
@aaupov  I'm ok with both namings, but just to be clear this function doesn't really create "function" it only create instructions to make return, creation of function it self is outside of this method, so maybe there is logic to rename it to "createDummyReturn". Or even "createReturn", what is dummy about that IDK :)

Reply:
Agree but let's keep renaming separate. Can rename as follow-up NFC change

Reply:
Thanks for your review. Done.

@yota9 yes, that was my intuition; I found implying that a function was created somewhat misleading.
`createReturn` is already used though, so we might go with something like: `createReturnBody`.
I will put up a follow-up NFC patch soon.